thirdparty/imgui/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.24 FATAL_ERROR)
+# Create the imgui libraries here since the submodule doesn't contain a CMakeLists.txt
+
+set(IMGUI_SOURCES
+  imgui/imgui_draw.cpp
+  imgui/imgui.cpp
+  imgui/imgui_demo.cpp
+  imgui/imgui_draw.cpp
+  imgui/imgui_tables.cpp
+  imgui/imgui_widgets.cpp
+  imgui/backends/imgui_impl_sdl2.cpp
+  imgui_impl_opengl3.cpp
```

Comment:
Add a `TODO: implement this with cmake magic, copy the file directly from imgui repo & patch it.`